AUTHORS

```diff
@@ -0,0 +1,19 @@
+The Prometheus project was started by Matt T. Proud in 2012.
```

Comment:
I can haz fame? How about:

"The Prometheus project was started by Matt T. Proud and Julius Volz in 2012."

To be fair, that doesn't point out precisely that Matt had some very early code sketches (mostly client lib) before SC.

Or maybe:

"The Prometheus project was initially started (conceived?) by Matt T. Proud and then co-created together with Julius Volz in 2012."